systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -415,6 +415,11 @@ void testSendMessagesPlainScramSha() {
                             .withPlain(listenerTls)
                         .endListeners()
                     .endKafka()
+                    .editEntityOperator()
+                        .editUserOperator()
+                            .withImage("scholzj/user-operator:latest")
```

Comment:
why the master should use a personal image? I guess it's a typo right?

Reply:
Dammit ... that should have been never committed ;-)